input/fsh/omrs-group.fsh

```diff
@@ -0,0 +1,36 @@
+/** OMRS Group resource  profile*/
+
+Profile: OMRSGroup
+Parent: Group
+Id: omrs-group
+Title: "OMRS Group"
+Description: "A Group Resource is described as Cohort in Fhir2 OMRS"
+* identifier 1..1
```

Comment:
I don't think the fhir2 module supports `identifier`, `managingEntity`, or `characteristic`, but we do have a description extension.

Reply:
Thank you @pmanko True I have checked identifier and characteristic are not supported by fhir2 module but for managingEntity was referenced in one of the test cases https://github.com/openmrs/openmrs-module-fhir2/blob/d0b34f74c243b32669d86c9369f7b226a0ada2f9/api-2.1/src/test/java/org/openmrs/module/fhir2/api/translators/impl/GroupTranslatorImpl_2_1Test.java#L223

Reply:
I think also  its being consumed by the baseGroupTranslastor https://github.com/openmrs/openmrs-module-fhir2/blob/d0b34f74c243b32669d86c9369f7b226a0ada2f9/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/BaseGroupTranslator.java#L73